docs/docs/projects/add-project.md

```diff
@@ -4,8 +4,8 @@ intro: Add one or multiple projects to group your infrastructure resources in a
 links:
     overview:
     quickstart:
-    previous: index
-    next: environments/add-environment
+    previous: pipelines/steps/remove-step
+    next: projects/list-projects
```

Comment:
To access `add ...` the user already went though the list, very likely, right?
Shouldn't the documentation lead users to take actions that help them move forward faster and get their environments setup, instead of explaining what they might have already found out by themselves?

See similar discussion at https://github.com/devopness/devopness/pull/1904#discussion_r2169835299